app/javascript/dashboard/store/modules/auth.js

```diff
@@ -143,6 +143,16 @@ export const actions = {
     }
   },
 
+  // eslint-disable-next-line no-empty-pattern
+  setActiveAccount: async ({}, { accountId }) => {
+    // eslint-disable-next-line no-useless-catch
+    try {
```

Comment:
Please do not disable the ESLint rule like this and I don't think there is any need to disable these here.